addons/binding/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/handler/MiIoAbstractHandler.java

```diff
@@ -342,4 +357,68 @@ protected JsonObject getJsonResultHelper(String response) {
         }
         return null;
     }
+
+    private boolean buildChannelStructure(String deviceName) {
```

Comment:
Why moving this to to abstract handler.
E.g the vacuumhandler will not need this at all.

My intention is to have the basic handler for the devices that follow the basic approach properties via getprop command and simple actions.
More special devices like the vacuum have their own handler extending the abstract handler